.github/workflows/link-check.yml

```diff
@@ -31,3 +33,19 @@ jobs:
           config-file: ".docs_mlc_config.json"
           folder-path: "docs"
           max-depth: 2
+  website-link-checker:
+    runs-on: ubuntu-latest
+    name: broken-link-check
+    steps:
+      - name: Check for broken links
+        id: link-report
+        uses: celinekurpershoek/link-checker@master
+        with:
+          # Required:
+          url: "https://www.featureform.com/"
+          # optional:
+          honorRobotExclusions: false
+          ignorePatterns: "*linkedin*"
+          recursiveLinks: true
+      - name: Print result
+        run: echo "${{steps.link-report.outputs.result}}"
```

Comment:
Will this just quitely echo the broken links or make it clear to us in some other way?
As in, do we have to keep checking this job?

Reply:
The job fails if there are any broken links, so it would be obvious from the outset